.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
```

Comment:
I think this works here because "Breaking Change" is supplemental to fix and feature. I would ideally like to see us only allow someone to select one.

Our guidance is "one PR per issue". This seems to support fixes and features going in together...which seems odd. Apologies if I missed this in a previous PR

Reply:
What are suggesting?